src/main/java/com/example/tournamentMaker/randomResultGenerator/RandomResultService.java

```diff
@@ -0,0 +1,186 @@
+package com.example.tournamentMaker.randomResultGenerator;
+
+import com.example.tournamentMaker.constans.Constans;
+import com.example.tournamentMaker.statistics.FootballStatistics;
+import com.example.tournamentMaker.statistics.FootballStatisticsRepository;
+import com.example.tournamentMaker.statistics.MatchResult;
+import com.example.tournamentMaker.team.Team;
+import com.example.tournamentMaker.team.TeamRepository;
+import com.example.tournamentMaker.team.player.FootballPlayer;
+import com.example.tournamentMaker.team.player.FootballPosition;
+import com.example.tournamentMaker.tournament.Tournament;
+import com.example.tournamentMaker.tournament.TournamentRepository;
+import com.example.tournamentMaker.tournament.enums.TournamentType;
+import com.example.tournamentMaker.tournament.game.Game;
+import com.example.tournamentMaker.tournament.game.GameRepository;
+import com.example.tournamentMaker.tournament.result.ResultService;
+import com.example.tournamentMaker.tournament.round.CupRoundService;
+import com.example.tournamentMaker.tournament.round.Round;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@Service
+@RequiredArgsConstructor
+class RandomResultService {
+    private final TournamentRepository tournamentRepository;
+    private final GameRepository gameRepository;
+    private final FootballStatisticsRepository footballStatisticsRepository;
+    private final ResultService resultService;
+    private final CupRoundService cupRoundService;
+    private final TeamRepository teamRepository;
+    private static final int MIN_YELLOW_CARDS_IN_MATCH = 0;
+    private static final int MAX_YELLOW_CARDS_IN_MATCH = 3;
+    private final Logger logger = LoggerFactory.getLogger(RandomResultService.class);
+
+    public void drawLotRoundsResults(RandomResultRequest request) {
+        Tournament tournament = tournamentRepository.findByName(request.getTournamentName())
+                .orElseThrow(() -> {
+                    throw new NoSuchElementException(Constans.NO_TOURNAMENT_FOUND);
+                });
+        String[] rounds = request.getRoundsToDraw().split("-");
+        int firstRound = Integer.parseInt(rounds[0]);
+        int lastRound = Integer.parseInt(rounds[1]);
+
```

Comment:
zawsze sa tylko dwie rundy? jezeli tak to moze istnieje lepszy sposob niz splitowac to po myslniku i brac 0 i 1

Reply:
Nie ma dwóch rund. Dostajemy stringa "3-6" co oznacza, że mamy wylosować wyniki dla rundy 3,4,5,6
Ale zawsze jest pierwsza i ostatnia